### ISSUE TITLE
<label> needed on <select> for Section 508 compliance

### ISSUE THREAD

**Issue by nutjob4life** [eea/eea.facetednavigation#210]
In the "Results per page" widget, the `<select>` generated does not have a matching `<label>` which is required by [WCAG section 1.1.1 "Non-Text Content"](https://www.w3.org/TR/WCAG21/#non-text-content). This is being flagged in a [Section 508](https://www.section508.gov/) scan of a site that uses EEA Faceted Navigation.

In the generated HTML below, there is no `<label>` for the `<select>` with ID = `c0`:
```html
<form action"." method="GET">
    <select class="faceted_select" name="c0" id="c0">
        <option value="0" title="0">0</option>
        <option selected="selected" value="20" title="20">20</option>
        <option value="40" title="40">40</option>
        …
    </select>
</form>
```

Yes, US Government websites have some pretty strict requirements 😰

**Comment by avoinea** [comment]
@nutjob4life Thank you for reporting this. A pull-request on the issue would be appreciated :wink:
